packages/cli/package.json

```diff
@@ -28,7 +28,7 @@
     "precommit": "yarn docs && git add docs README.md ../../docs",
     "version": "yarn docs && git add docs/* README.md",
     "postversion": "git push && git push --tags",
-    "lint": "eslint src snippets --fix",
+    "lint": "eslint src snippets",
```

Comment:
Remove `--fix` so eslint returns a non-zero value instead of actually fixing it when there's a lint error.

Reply:
Ah, this is interesting! If we're running `yarn lint` during development then we'd probably want `--fix` included. But, I almost never do that (my editor auto-fixes what it can, as does the pre-commit hook). Given that this is used in CI where we _do_ want it to fail, this is probably fine.